src/app/shared/menu/date/custom-date-range-dropdown.component.ts

```diff
@@ -10,7 +10,10 @@ import { GoogleAnalyticsEventAction, RouteAdvancedRangeAction } from '../../../n
     <div class="dropdown" [class.open]="open">
       <div class="overlay" (click)="toggleOpen()"></div>
       <div class="dropdown-button">
-        <button class="btn-icon icon-calendar grey-dove" (click)="toggleOpen()" aria-label="Custom Date Range"></button>
+        <button (click)="toggleOpen()">
+          <metrics-date-range-summary [routerState]="routerState"></metrics-date-range-summary>
```

Comment:
I'm noting this component has Hourly/Daily/Weekly/Monthly buttons in it.  Are those necessary, since the dropdown for those is right next to this button?

Reply:
We didn't have any changes to the content of the desktop dropdowns in the designs, so I'm not sure if we meant to keep the quick interval selector and have the advanced all option selector on desktop, or what, but this is how I interpreted it.